build/virt-prerunner/cloud-hypervisor-type

```diff
@@ -0,0 +1 @@
+longrun
```

Comment:
What if CH crashed? IIUC, type _longruns_ get supervised by s6 and will be restarted if crashed, pod `prerunner` will never get failed.

Currently, errors will be reported in `daemon` controller when VM info not found for a `Running` VM, but i noticed there is a `TODO` that means VM info not found will be ignored in the future.
https://github.com/smartxworks/virtink/blob/ff9679140b2b1b6d298d8eec581b27258b222111/pkg/daemon/vm_controller.go#L97-L101

Reply:
as s6 adviced, I add a [finish script](https://github.com/just-containers/s6-overlay#writing-an-optional-finish-script) for cloud-hypervisor service, when it stoped, the finish script will be executed, and in finish script call `/run/s6/basedir/bin/halt` to stop the container.